kore/app/share/GlobalMain.hs

```diff
@@ -581,14 +582,15 @@ execute options metadataTools lemmas worker =
             (declareSMTLemmas metadataTools lemmas)
             worker
     withoutSMT = SMT.runNoSMT worker
-    KoreSolverOptions{timeOut, rLimit, resetInterval, prelude, solver} =
+    KoreSolverOptions{timeOut, rLimit, resetInterval, prelude, solver, args} =
         options
     config =
         SMT.defaultConfig
             { SMT.timeOut = timeOut
             , SMT.rLimit = rLimit
             , SMT.resetInterval = resetInterval
             , SMT.prelude = prelude
+            , SMT.arguments = args <> SMT.Config.arguments SMT.defaultConfig
```

Comment:
Maybe we should only use the default if the `args` are empty? This would enable users to get rid of defaults that we may set in the future.
```suggestion
            , SMT.arguments = if null args then SMT.Config.arguments SMT.defaultConfig else args
```